concepticondata/conceptlists/Gruehn-2008-200.tsv

```diff
@@ -0,0 +1,201 @@
+Gruehn-2008-200-113	113	krank	ill	1847	SICK	21.402	2.167	1.833	1.917	2.417	2	2.167	2.083	4.667	4.75	4.667	4.417	4.708	4.542	4.625	1.917	1.5	3.25	3.333	1.708	3.292	2.5	6.25	6.333	5.167	6	6.292	5.583	5.938	2.167	1.75	2.667	3.167	1.958	2.917	2.438	4.5	4.333	4.167	4.333	4.417	4.25	4.333	1.563	2.563	1.958
+Gruehn-2008-200-114	114	kreativ	creative			4.052	6.25	5.917	6.417	6.167	6.083	6.292	6.188	2.917	2.583	3.083	4.25	2.75	3.667	3.208	5.167	4.5	5.167	5.417	4.833	5.292	5.063	5.667	5.083	5.917	4.667	5.375	5.292	5.333	5.167	5.5	5.667	4.818	5.333	5.261	5.298	2.167	1.833	2.5	2.182	2	2.348	2.17	6.313	6.188	1.75
+Gruehn-2008-200-115	115	kritisch	critical			20.751	4.25	4.417	5.25	5.5	4.333	5.375	4.854	5.333	5.083	5	5.167	5.208	5.083	5.146	5.75	5.083	5.25	5.667	5.417	5.458	5.438	2.917	3.25	3.75	3.417	3.083	3.583	3.333	5.667	5.417	6	5.5	5.542	5.75	5.646	2.75	2.833	3.25	3.636	2.792	3.435	3.106	5.625	5.438	1.5
+Gruehn-2008-200-116	116	lahm	lame	320	LAME	4.55	2.083	2.417	2.417	2.167	2.25	2.292	2.271	3	3	2.75	2.75	3	2.75	2.875	2.833	2.083	2.167	2.083	2.458	2.125	2.292	4.75	4.545	4.833	5.417	4.652	5.125	4.894	1.5	2.5	1.917	1.833	2	1.875	1.938	3.667	4.167	3.667	3.917	3.917	3.792	3.854	1.75	1.875	1.75
+Gruehn-2008-200-117	117	langsam	slowly	701	SLOW	62.801	3	3.25	2.917	3	3.125	2.958	3.042	2.417	2.833	2.333	2.583	2.625	2.458	2.542	3.833	3.5	3.167	3.583	3.667	3.375	3.521	5.167	5.083	5.583	4.417	5.125	5	5.063	2.167	3.25	2.583	2.417	2.708	2.5	2.604	4.25	4.333	3.917	3.917	4.292	3.917	4.104	2.5	2.375	1.833
+Gruehn-2008-200-118	118	launisch	moody			628	2.083	2.083	1.583	1.917	2.083	1.75	1.917	5.5	5.167	5.5	5.083	5.333	5.292	5.313	2.583	2.25	3.417	2.333	2.417	2.875	2.646	4.417	3.25	4.455	3.727	3.833	4.091	3.957	3.25	3.417	1.75	2.5	3.333	2.125	2.729	2.917	3	2.917	3.667	2.958	3.292	3.125	1.875	1.75	1.208
+Gruehn-2008-200-119	119	lebhaft	lively			5.408	6.083	5.917	6.083	5.5	6	5.792	5.896	3.917	3.833	5.083	4.818	3.875	4.957	4.404	4.833	5.083	4.5	4.083	4.958	4.292	4.625	5.5	5.333	5.583	5.583	5.417	5.583	5.5	5.917	5.667	6	4.333	5.792	5.167	5.479	1.75	1.364	1.917	1.833	1.565	1.875	1.723	5.938	5.625	1.667
+Gruehn-2008-200-120	120	liberal	liberal			2.883	5.333	4.417	6.25	5.167	4.875	5.708	5.292	2.667	2.75	2.667	3.25	2.708	2.958	2.833	4.25	4.25	4	4.667	4.25	4.333	4.292	2.167	2.333	3.25	2.417	2.25	2.833	2.542	4.667	4.25	5.417	5.083	4.458	5.25	4.854	2.833	2.667	3.417	3.417	2.75	3.417	3.083	5.5	5.5	1.292
+Gruehn-2008-200-121	121	listig	wily			1.397	3	2.833	3.167	2.5	2.917	2.833	2.875	5	5.167	5.417	5.273	5.083	5.348	5.213	6	5.583	5.167	5.583	5.792	5.375	5.583	4	5.083	3.917	4	4.542	3.958	4.25	3	3.333	1.333	2	3.167	1.667	2.417	2.917	2.417	3.083	3.083	2.667	3.083	2.875	3.625	2.625	1.25
+Gruehn-2008-200-122	122	lustig	jovial			17.447	6.417	6.5	6	5.917	6.458	5.958	6.208	2.583	2.083	2.5	3.083	2.333	2.792	2.563	4.667	4.25	3.667	4.333	4.458	4	4.229	6.167	6.25	6.167	5.364	6.208	5.783	6	5.667	5.833	5.25	4.833	5.75	5.042	5.396	2.667	2.333	2.5	2.25	2.5	2.375	2.438	6.063	6.063	1.708
```

Comment:
Hm, we've got _lustig_ mapped to `MERRY` as well. Have to check this as well.

Reply:
I think we should discuss the mappings of MERRY separately because there should not be too many double mappings in this list.

Reply:
Agreed.